.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
-          ruby-version: "2.7"
+          ruby-version: "3.4"
```

Comment:
Should https://github.com/heroku/heroku-buildpack-scala/blob/db9ee902854336f2ad617c98e7400a8da365998b/.github/workflows/hatchet_app_cleaner.yml#L27 be updated too?

Reply:
Good spot! Thanks!